pkg/api/server.go

```diff
@@ -68,7 +71,7 @@ type ApiServer struct {
 }
 
 func (a *ApiServer) Start() error {
-	docs.SwaggerInfo.Version = "0.1"
+	docs.SwaggerInfo.Version = getDaytonaLatestReleaseTag()
```

Comment:
You can get the version from `internal.Version` from the codebase. That variable gets replaced at build time.